packages/abi-coder/src/coders/v1/enum.test.ts

```diff
@@ -0,0 +1,62 @@
+import { ErrorCode, FuelError } from '@fuel-ts/errors';
+import { expectToThrowFuelError } from '@fuel-ts/errors/test-utils';
+import { bn } from '@fuel-ts/math';
+
+import { U64_MAX } from '../../../test/utils/constants';
+import { U64Coder } from '../v0/u64';
+
+import { BooleanCoder } from './boolean';
+import { EnumCoder } from './enum';
+
+/**
+ * @group node
+ * @group browser
```

Comment:
Why do we need to run this on the browser?

Reply:
Good question. For me there isn't necessarily a need but if it can, and it is used in a browser environment (unlike CLI tools etc), then we should test it in a browser env.

Reply:
Yes, the default rationale should be to test everything in the browser.

Unless there's a reason not to, which is the case for CLI tools.